submissions/train-stop-limits/train-stop-limits.md

```diff
@@ -0,0 +1,41 @@
+
+{% include image.html src='https://cdn.factorio.com/assets/img/blog/fff-361-train-stop-limit.png' alt='Train Stop Limit GUI' caption='Source: <a href="https://www.factorio.com/blog/post/fff-361">FFF-361</a>' %}
+
+### What is TSM and why should I care?
+
+Train supply manager, a.k.a. train pub sub, is a mod which allow train stops to request a train when certain circuit conditions are true, for example when less than a certain number of trains are enroute to the stop. It also allows you do more complex circuit wizardry such as only requesting a train when there is enough material to actually fill the train, though I have never used this functionality.
+
+The dream for TSM, is to fully implement a [just in time](https://en.wikipedia.org/wiki/Just-in-time_manufacturing) system, by ensuring trains only ever travel full and only when they are explicitly needed. This allows you to massively reduce the number of trains on a given network by removing only send trains to be filled when there is enough resources at a given outpost and only send them to drop off when the resources in the given area are low. At all other times, the trains are sat stationary, causing zero congestion. In this system, the trains are requested at both the collection and drop off stops, This is especially important for larger bases and is the reason I love TSM so much.
+
+{% include video.html mp4='https://media.alt-f4.blog/ALTF4/15/TSL-Video-1.mp4' width='1000px' height='900px' alt='Video of TSM in action' caption='In the video, the loading and unloading stop each only ever have one train stopped at them ensuring no backlogs. When a stop becomes empty, a train is dispatched from the depot by TSM to refill the stop which most urgently needs it.' %}
+
+### Can Train Stop Limits replace TSM?
+
+To understand when TSM will be needed vs just using the train stop limit, I recreated the same testing TSM setup in vanilla 1.1 with [great sucess](https://www.youtube.com/watch?v=J88-RdWnNT0)! 
+
+{% include video.html mp4='https://media.alt-f4.blog/ALTF4/15/TSL-Video-2.mp4' width='1000px' height='900px' alt='Video of same setup using train stop limits' caption='This video shows the same setup of copper trains load and unloading, again with only ever one train at the loading and unloading stops, whilst the others wait at the depot.' %}
+
+The reason this works so well and is such a good drop in replacement for TSM is that the trains wait at the depot stop until there is space. If the train were to wait at the loading spot, this would cause congestion and limit the output capacity of the producer. Using depot stops allows the trains to have a "staging area" of sorts, to wait until they are needed, out of the way of the train network.
```

Comment:
I think this can also be a little clearer with terminology. For example: 
- ` at the depot`: I had to think about this term since I don't typically play with train depots. Not sure what to call it though. You explain it later in the paragraph, but it might be worth it to explain it first.
- `until there is space` in the loading station.